includes/elementor-functions.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+use Elementor\Plugin;
+use Elementor\Core\Kits\Documents\Kit;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Register Site Settings Controls.
+ */
+
+add_action( 'elementor/init', 'hello_elementor_settings_init' );
+
+function hello_elementor_settings_init() {
+	if ( hello_header_footer_experiment_active() ) {
+		require 'settings/settings-header.php';
+		require 'settings/settings-footer.php';
+
+		add_action( 'elementor/kit/register_tabs', function( Kit $kit ) {
+			$kit->register_tab( 'hello-settings-header', HelloElementor\Includes\Controls\Settings_Header::class );
+			$kit->register_tab( 'hello-settings-footer', HelloElementor\Includes\Controls\Settings_Footer::class );
```

Comment:
Make sure the namespaces are updated here too.

Reply:
Done.